app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  before_action :lesson_find, only: [:create, :edit, :update]
+
+ def create
+    @comment = current_user.comments.create(comment_params.merge(lesson: @lesson)
```

Comment:
There is no ) in the end of the line. Production will be broken.
@evrusha I think, we need to wait until your PR with Rollbar will be merged, cause fix already there.

Reply:
Rollbar merged to development branch